lightningd/lightningd.c

```diff
@@ -482,5 +699,7 @@ int main(int argc, char *argv[])
 	memleak_cleanup();
 #endif
 	daemon_shutdown();
+
+	/*~ Farewell.  Next stop: hsmd/hsm.c. */
```

Comment:
why is it actually `lightningd.c` with a d but `hsm.c` without d ?

Reply:
Good point.  That's bugged me for a while too, so I renamed them all...